migrations/v2.js

```diff
@@ -0,0 +1,182 @@
+    return true;
+  });
+
+  /**
+   * * Add field to each item in a JSON array and set blank.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._items._graphic.src attribute', async () => {
+    accordions.forEach(accordion => {
+      accordion._items.forEach(item => {
+        item._graphic.src = '';
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check add accordion._items._graphic.src atrribute', async () => {
+    const isValid = accordions.every(accordion =>
+      accordion._items.every(item =>
+        item._graphic && item._graphic.src !== undefined
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-accordion - _graphic.src not added to every instance of accordion._items');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v2.0.0', { name: 'adapt-contrib-accordion', version: '2.0.0', framework: '2.0.0' });
+});
+
+describe('adapt-contrib-accordion - v2.0.0 > v2.0.4', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.0.0', { name: 'adapt-contrib-accordion', version: '<=2.0.4' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+  * * Add JSON field to component and set blank.
+  */
+  mutateContent('adapt-contrib-accordion - add accordion._classes', async () => {
+    accordions.forEach(accordion => {
+      accordion._classes = '';
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._classes atrribute', async () => {
+    const isValid = accordions.every(({ _classes }) => _classes !== undefined);
+    if (!isValid) throw new Error('adapt-contrib-accordion - _classes not added to every instance of accordion');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v2.0.4', { name: 'adapt-contrib-accordion', version: '2.0.4', framework: '>=2' });
+});
+
+describe('adapt-contrib-accordion - v2.0.4 > v2.0.5', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.0.4', { name: 'adapt-contrib-accordion', version: '<=2.0.5' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add JSON field to component and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._shouldCollapseItems', async () => {
+    accordions.forEach(accordion => {
+      accordion._shouldCollapseItems = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._shouldCollapseItems atrribute', async () => {
+    const isValid = accordions.every(({ _classes }) => _classes !== undefined);
```

Comment:
copy and paste error - `_classes` should be `_shouldCollapseItems`

Reply:
Fixed.